src/CoreBundle/Resources/public/javascript/cms.js

```diff
@@ -57,31 +57,41 @@ function GetAjaxCallTransparent(url, functionName) {
 }
 
 function AjaxError(XMLHttpRequest, textStatus, errorThrown) {
-    if (textStatus === 'parsererror') {
-        window.parent.$.unblockUI();
-        $.unblockUI();
-        toasterMessage('Error! Wasn`t able to parse ajax response.', 'ERROR');
-        if (XMLHttpRequest.responseText !== '') {
-            var sError = XMLHttpRequest.responseText;
+    window.parent.$.unblockUI();
+    $.unblockUI();
 
-            if (sError.length > 1024) {
-                sError = sError.substr(0, 1024);
-            }
+    var errorMessage = "Error";
+
+    if (XMLHttpRequest.responseText !== "" || XMLHttpRequest.statusText !== "") {
+        var responseError = XMLHttpRequest.responseText;
 
-            if (sError.indexOf('<title>') !== -1) sError = '';
+        if (responseError.length > 1024) {
+            responseError = responseError.substr(0, 1024);
+        }
 
-            var sMessage = CHAMELEON.CORE.i18n.Translate('chameleon_system_core.js.ajax_error', sError);
+        if (responseError.indexOf('<title>') !== -1) {
+            responseError = XMLHttpRequest.statusText;
+        }
 
+        errorMessage = CHAMELEON.CORE.i18n.Translate('chameleon_system_core.js.ajax_error', responseError);
+    }
+
+    if (textStatus === 'parsererror') {
+        toasterMessage('Error! Wasn`t able to parse ajax response.', 'ERROR');
```

Comment:
Could we take the opportunity to add translation for this error?

And I'd prefer a single quote instead of the backtick in "wasn't" - backticks are technically a bit unexpected and I think we use quotes everywhere.

Reply:
ok